submissions/ivan-chukhalo/pop-up_hooli/index.html

```diff
@@ -0,0 +1,112 @@
+                </li>
+                <li class="popup__item">
+                  <a href="#" class="popup__link link icon_yellow">yellow</a>
+                </li>
+                <li class="popup__item">
+                  <a href="#" class="popup__link link icon_drive">drive</a>
+                </li>
+                <li class="popup__item">
+                  <a href="#" class="popup__link link icon_flower">flower</a>
+                </li>
+                <li class="popup__item">
+                  <a href="#" class="popup__link link icon_desk">desk</a>
+                </li>
+                <li class="popup__item">
+                  <a href="#" class="popup__link link icon_calendar">calendar</a>
+                </li>
+                <li class="popup__item">
+                  <a href="#" class="popup__link link icon_musiem">musiem</a>
+                </li>
+                <li class="popup__item">
+                  <a href="#" class="popup__link link icon_apps">apps</a>
+                </li>
+                <li class="popup__item">
+                  <a href="#" class="popup__link link icon_excel">tables</a>
+                </li>
+                <li class="popup__item popup__item_more">
+                    <input type="checkbox" class="checkbox checkbox_more" id="checkbox_more" />
+                    <label class="popup__item popup__more" for="checkbox_more">More</label>
+                  <div class="extra-wrapper">
+                    <ul class="extra-items">
+                      <li class="popup__item">
+                        <a href="#" class="popup__link link icon_accaunt">account</a>
+                      </li>
+                      <li class="popup__item">
+                        <a href="#" class="popup__link link icon_gmail">email</a>
+                      </li>  
+                      <li class="popup__item">
+                        <a href="#" class="popup__link link icon_light">light</a>
+                      </li>
+                      <li class="popup__item">
+                        <a href="#" class="popup__link link icon_infinity">infinity</a>
+                      </li>
+                      <li class="popup__item">
+                        <a href="#" class="popup__link link icon_maps">geo</a>
+                      </li>
+                      <li class="popup__item">
+                        <a href="#" class="popup__link link icon_card">card</a>
+                      </li>
+                    </ul>
+                </div>  
+                </li>
+              </ul>
+          </div>
+          </li>
+          <li class="nav__list-item">
+            <a href="#" class="nav__link">
+              <img src="./assets/icons/icon_bell.png" alt="" class="nav__icon" />
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a href="#" class="nav__link">
+              <img src="./assets/avatar.png" alt="Go to the account settings" class="nav__avatar" />
```

Comment:
What about that icon?

Reply:
@A-Ostrovnyy okay. Should I also get rid of alt value of avatar image? I've set it because I'd thought it should be for a screenreader and users with limited abilities.

Reply:
If you want to provide accessibility you should add `aria-label` to the link)